### ISSUE TITLE
Limit Console Output always scrolls to the bottom

### ISSUE THREAD

**Issue by adamconroy** [rstudio/rstudio#8552]
### System details

    RStudio Edition : <Desktop or Server
    RStudio Version : 1.4.X
    OS Version      : 
    R Version       : 

### Steps to reproduce the problem

1. Turn on Limit Console Output
2. Create a function that outputs constantly to the console
3. Observe that the console will always scroll to the bottom

### Describe the problem in detail

It's difficult to navigate around the console while it continues to output because it always scrolls to the bottom.

I spent a day investigating this and the cause isn't what I thought it was. I was originally doing this on purpose and thought that I could just comment this line out and we'd be good: https://github.com/rstudio/rstudio/blob/eca45fbb0a356667241d406fd82c529b598c0347/src/gwt/src/org/rstudio/core/client/virtualscroller/virtualscroller.js#L268

But it is not the case. Something else is scrolling it to the bottom. I tried commented out pretty much everything in virtualscroller.js and wasn't able to find the cause in a few hours so going to just file this issue and put it on Juliet-Rose.

### Describe the behavior you expected

If you're scrolled up it doesn't auto scroll to the bottom when new content is appended to the console.

**Comment by valerie-rstudio** [comment]
### Verify Failed

    RStudio Edition : Desktop
    RStudio Version : 1.4.1527
    OS Version      : mac OS 11.1
    R Version       : 4.0.2

### Failure Points
When a long running function completes its final output, the console scrolls to the bottom.
1. Spam the console with any input until you can scroll.
2. Run this R code `for (x in 1:5)
{
  print(x)
  Sys.sleep(5)
}`
3. While the above code is running, scroll all the way up on the console.
4. **Expected Result:** The console remains in the same scroll position.
5. **Actual Result:** The console scrolls to the bottom once the code is finished.

### Points Verified
- When a long running function outputs to the console, and it is not the final output of that function, then the console maintains the users scroll position.

**Comment by adamconroy** [comment]
@valerie-rstudio I believe that we are properly maintaining the functionality without the option turned on. In the testing I just did now it only scrolls to the bottom when the long running command starts or stops but it doesn't scroll as output is added during the `for` loop. This is the same functionality as without the "limit output" option turned on so I think we're in the clear here. Can you double check that?

**Comment by kevinushey** [comment]
There are two components to the behavior to be verified here:

1. If the user has scrolled console history to the bottom, then it should remain at the bottom, even as new output is streamed in.

2. If the user begins scrolling the console history even while output is streaming in, the console should not scroll back to the bottom.

In other words, "by default" users should see any console output as it is emitted; if they start scrolling the console history, then that "stay scrolled to bottom" behavior should switch off.

In my case, with "Limit Visible Console Output" turned off, I no longer see the behavior described in (1).

Note that after changing the "Limit Visible Console Output" preference I think you need to restart RStudio as well. (we should probably prompt in the UI when a user has changed an option requiring restart via the command palette)

**Comment by adamconroy** [comment]
(1) is captured in this issue: https://github.com/rstudio/rstudio/issues/8504 (the name of the issue is poor). I've confirmed that this behavior is extra unreliable on smaller screens and desktop and am working on fixing it now.

(2) I believe that the behavior has not regressed, Valerie is just seeing the jump after output finishes which I could see being off-putting to a user as well but it is what happens with Limit Output turned off as well so is currently expected.

**Comment by adamconroy** [comment]
Putting test back on this to force it back into close process. #8504 is redundant with what is being recorded as still being seen here.

**Comment by valerie-rstudio** [comment]
### DUPLICATE

Closing as duplicate of issue [#8504](https://github.com/rstudio/rstudio/issues/8504)
